frontend/src/utils/fetchTechnicalProgression.js

```diff
@@ -0,0 +1,10 @@
+export default async function fetchTechnicalProgression(productCode) {
```

Comment:
can you add documentation for the t levels the same way we've done with the other utils?